app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -241,6 +254,14 @@ def index_organization
       return subscriptions
     end
 
+    def consumed_subscriptions(consumer, organization = current_organization)
+      entitlements = consumer.consumed_entitlements
+      entitlements.delete_if do |entitlement|
+        pool = Pool.find_pool(entitlement.poolId)
+        !pool.provider?(organization)
+      end
```

Comment:
yes I don't either, it looks like I don't have any need for it after refactoring to use the url params, I'll remove